desc/objectives/linear_objectives.py

```diff
@@ -38,9 +38,28 @@ def update_target(self, eq):
         if self._use_jit:
             self.jit()
 
+    def _parse_target_from_user(
+        self, target_from_user, default_target, default_bounds, idx
+    ):
+        if target_from_user is None:
+            target = default_target
+            bounds = default_bounds
+        elif isinstance(target_from_user, tuple) and (
+            len(target_from_user) == 2
+        ):  # treat as bounds
+            target = None
+            bounds = (
```

Comment:
Why do you broadcast bounds to a different shape? I guess it's needed for somewhere else?

Reply:
its because bounds is a tuple of two arrays (lower, upper), rather than a single array. Each bound should have the same shape as target, which is the same as dim_f